src/user-guide/running-workflows/simulation-modes.rst

```diff
@@ -41,18 +41,62 @@ If :cylc:conf:`[runtime][<namespace>]execution time limit` and
 run length is computed by dividing the time limit by the speedup factor.
 
 
+Simulated Failure
+^^^^^^^^^^^^^^^^^
+
+Tasks always complete custom outputs.
+
+By default they succeed, and return a succeeded output.
+
+.. warning::
+
+   In simulation mode a succeeded output may not imply that
+   submitted and/or started outputs are generated - so you will not
+   be able to test graph pathways such as ``task:started => do_something``.
+
+You can set some or all instances of a task to fail using
+:cylc:conf:`[runtime][<namespace>][simulation]fail cycle points`.
+``fail cycle points`` takes either a list of cycle point strings or "all".
+
+Tasks set to fail will succeed on their second or following simulated
+submission. If you want all submissions to fail, set
+:cylc:conf:`[runtime][<namespace>][simulation]fail try 1 only = False`.
+
+For example, to simulate a task you know to be flaky on the half
+hour but not on the hour:
+
+.. code-block:: cylc
+
+   [[get_observations]]
+      execution retry delays = PT30S
+      [[[simulation]]]
+         fail cycle points = 2022-01-01T00:30Z,  2022-01-01T01:30Z
+
+In another case you might not expect the retry to work, and want to test
+whether your failure handling works correctly:
+
+.. code-block:: cylc
+
+   [[get_data]]
+       execution retry delays = PT30S
```

Comment:
Does this work?

Reply:
If by "work" you mean "fails multiple times", then it works. Although thinking about it it's a slightly silly example because the `server_broken` output will appear at the same time as the first failure.